ros_ws/src/control/launch/robot.launch

```diff
@@ -16,6 +16,7 @@
     
     <!-- Load parameter file -->
     <rosparam command="load" file="$(find control)/param/igvc.yaml"/>
+    <rosparam command="load" file="$(find control)/param/controllers/moga.yaml"/>
```

Comment:
It would be nice if we could pick which controller to use with a command line argument.  We could use the xbox as the default but use a different controller file if they pass it when launching.  More info here: http://wiki.ros.org/roslaunch/XML/arg